packages/bottles-v1-deb/bottles-v1-deb.pacscript

```diff
@@ -0,0 +1,8 @@
+name="bottles-v1-deb"
+gives="bottles-v1"
```

Comment:
`gives` does not utilize the actual deb name, which is `com.github.mirkobrombin.bottles`